metrics/main.go

```diff
@@ -18,7 +18,8 @@ import (
 var (
 	projectID             = flag.String("project_id", "bazel-untrusted", "ID of the GCP project.")
 	datastoreSettingsName = flag.String("datastore_settings_name", "MetricSettings", "Name of the settings entity in Datastore.")
-	testMode              = flag.Bool("test", false, "If true, the service will collect and publish all metrics immediately and only once.")
+	testMode              = flag.Bool("test", false, "If true, the service will collect and publish all metrics to stdout immediately and only once. No metric will be published to CloudSQL or Stackdriver.")
+	testStackdriver       = flag.Bool("always_enable_stackdriver", false, "If true, all metrics that normally publish to Stackdriver will do so even if test mode is enabled.")
```

Comment:
Sometimes we need to test the Stackdriver integration, and in these cases we use the flag (since this PR also disables Stackdriver publishing in test mode by default).